src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -165,6 +165,12 @@ const registry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Con
 				'minimum': 38,
 				'markdownDescription': localize({ comment: ['This is the description for a setting. Values surrounded by single quotes are not to be translated.'], key: 'workbench.editor.tabSizingFixedMaxWidth' }, "Controls the maximum width of tabs when `#workbench.editor.tabSizing#` size is set to `fixed`.")
 			},
+			'workbench.editor.tabHeight': {
```

Comment:
I am starting to wonder why this setting could not apply when tabs are disabled. Here is how it would look like, tweaking the `35px` value in `notabstitlecontrol.css`:

<img width="626" alt="image" src="https://github.com/microsoft/vscode/assets/900690/17b5d98f-bd76-4527-ac68-94052f139a6d">

I think this could work even in that case and then this setting could become more generic for both tabs and no-tabs.

Reply:
Also, I wonder if the `small` property should be `compact`? This would already align with https://github.com/microsoft/vscode/issues/168671 if we ever had it.